frontend/src/components/Signup/Signup.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import './Signup.css'
+
+const Signup = () => {
+  return (
+
+    <div className="container">
+
+      <div className="header">
+        <div className="logo">[Permuto Logo]</div>
+        </div>
+
+        <h2>CREATE ACCOUNT</h2>
+
+      <div className="signup-form">
+
+        <form>
+          <label htmlFor="email">EMAIL</label>
+          <input type="text" id="email" name="email" placeholder="Text" />
+          
+          <label htmlFor="password">PASSWORD</label>
+          <input type="password" id="password" name="password" placeholder="Text" />
+
+          <label htmlFor="password">CONFIRM PASSWORD</label>
+          <input type="password" id="password" name="password" placeholder="Text" />

```

Comment:
The form has two password inputs with the same id attribute (id="password"). What you can do here is to change the confirm password input id to "confirmPassword"

```css
<label htmlFor="passwordConfirm">CONFIRM PASSWORD</label>
<input type="password" id="passwordConfirm" name="passwordConfirm" placeholder="Text" />
```